enterprise/backup/src/main/java/org/neo4j/backup/BackupService.java

```diff
@@ -217,11 +219,20 @@ private BackupOutcome incrementalBackup( FileSystemAbstraction fileSystem, Strin
 
         Map<String,String> temporaryDbConfig = getTemporaryDbConfig();
         config = config.with( temporaryDbConfig );
+
+        Map<String,String> configParams = new HashMap<>();
+        Set<String> keys = config.getConfiguredSettingKeys();
+        for ( String key : keys )
+        {
+            Optional<String> value = config.getRaw( key );
+            value.ifPresent( s -> configParams.put( key, s ) );
+        }
```

Comment:
This piece of logic is converting the Config object into a StringMap. Required because GraphDatabaseBuilder does not yet accept a Config object (see comment above about required breaking interface changes)